tokio/src/runtime/basic_scheduler.rs

```diff
@@ -188,7 +178,9 @@ where
                 scheduler.tick(local);
 
                 // Maintenance work
-                scheduler.drain_pending_drop();
+                unsafe {
```

Comment:
I love that these methods are now marked as `unsafe`. However, I _would_ then also like to see a comment on each added `unsafe` call to those methods arguing why the caller upholds the necessary safety invariants.

Reply:
added in 7d78ba66863e737fbb7009866c35e638e9251774, thanks for pointing that out!